zos_concepts/data_transfer/archive_copy_unarchive_restore/archive_fetch_data_sets.yml

```diff
@@ -0,0 +1,119 @@
+###############################################################################
+# © Copyright IBM Corporation 2023
+###############################################################################
+
+###############################################################################
+# This playbook demonstrates how to archive one or more z/OS data sets and
+# fetch the archive to the controller using Red Hat Ansible Certified Content
+# for IBM Z.
+#
+# Usage:
+#  ansible-playbook -i <inventory> <playbook>
+#
+# Example:
+#  ansible-playbook -i inventories archive_fetch_data_sets.yml
+#
+#
+# Requirements:
+#   IBM z/OS core collection 1.7.0 or later.
+#   Ansible Community General Collection
+#
+# Configure:
+#   data_sets_to_archive - A list of data sets contained in an archive you wish to include in dump.
+#   archive_data_set - The name to use for the data set archive,
+#                       if not provided, module-generated name will be used.
+#   delete - Indicates if terse and archive data sets should be deleted upon successful transfer.
+#
+###############################################################################
+
+- hosts: source
+  collections :
+    - "ibm.ibm_zos_core"
+  gather_facts: false
+  vars:
+    data_sets_to_archive: ["USER.PRIVATE.TEST"]
+    archive_data_set: ""
+    delete: true
+  environment: '{{ environment_vars }}'
+
+  tasks:
+    - name: Raise error if data_sets_to_archive is not set
+      ansible.builtin.fail:
+        msg: Missing required variable 'data_sets_to_archive'.
+      failed_when: data_sets_to_archive is not defined
```

Comment:
i think this task can be deleted now that `data_sets_to_archive` has a default definition in the playbook

Reply:
done